app/serializers/user.js

```diff
@@ -1,11 +1,15 @@
 import ApplicationSerializer from 'open-event-frontend/serializers/application';
-import { pick } from 'lodash';
+import { pick, omit } from 'lodash';
 
 export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (options && options.includeId) {
```

Comment:
Instead of all this, do 

```js
if (snapshot.id) { // id only exists for update. Not for create.
  let attributesToOmit = ['email'];
  if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
    attributesToOmit.push('password');
  }
  json.data.attributes = omit(json.data.attributes, omit);
} else {
  json.data.attributes = pick(json.data.attributes, ['email', 'password']);
}
```

So, by default, email and password are ignored for all updates. When we do want to update password (admin feature), at that time, we'll pass in `adapterOptions` in save accordingly.